server.js

```diff
@@ -11,13 +14,38 @@ const welcomeMessage = {
   text: "Welcome to CYF chat system!",
 };
 
+const PORT = 9002;
+
 //This array is our "data store".
 //We will start with one message in the array.
 //Note: messages will be lost when Glitch restarts our server.
-const messages = [welcomeMessage];
+let messages = [welcomeMessage];
+
+app
+  .get("/", function (request, response) {
+    response.sendFile(__dirname + "/index.html");
+  })
+  .post("/messages", function (request, response) {
+    const newMessage = request.body;
```

Comment:
don't forget the message should have a `from` property when it's stored in the `messages` array. How can you add this to the request body and use it in here?